images/win/toolset-2019.json

```diff
@@ -47,6 +47,16 @@
                 "12.*",
                 "14.*"
             ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x64",

```

Comment:
I don't see file with this architecture for Windows in manifest json

Reply:
Changed to correct version